tests/test_integration.py

```diff
@@ -0,0 +1,151 @@
+  
+## Importing these packages is specific for this simulation case
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import xarray as xr
+from mne.io import read_info
+from scipy.stats import gamma
+from pathlib import Path
+
+import hmp
+from hmp import simulations
+
+epoch_data_file = Path("tutorials", "sample_data", "sample_data.nc")
+info_data_file = Path("tutorials", "sample_data", "eeg", "processed_0022_epo.fif")
+
+def test_integration():
+    fig, ax = plt.subplots(1)#captures plots
+    cpus = 1 # For multiprocessing, usually a good idea to use multiple CPUs as long as you have enough RAM
+
+    n_trials = 2 #Mini for testing
+    sfreq = 100
+    n_events = 2
+    frequency = 10. #Frequency of the event defining its duration, half-sine of 10Hz = 50ms
+    amplitude = 1 #Amplitude of the event in nAm, defining signal to noise ratio
+    shape = 2 #shape of the gamma distribution
+    means = np.array([60, 150, 80])/shape #Mean duration of the between event times in ms
+    names = ['inferiortemporal-lh','caudalanteriorcingulate-rh','bankssts-lh']#Which source to activate for each event (see atlas when calling simulations.available_sources())
+
+    sources = []
+    for source in zip(names, means): #One source = one frequency, one amplitude and a given by-trial variability distribution
+        sources.append([source[0], frequency, amplitude, gamma(shape, scale=source[1])])
```

Comment:
```suggestion
    for cur_name, cur_mean in zip(names, means): #One source = one frequency, one amplitude and a given by-trial variability distribution
        sources.append([cur_name, frequency, amplitude, gamma(shape, scale=cur_mean)])
```
This would be my suggestion, but `cur_` might be changed to something more descriptive.

Reply:
Thanks, I implemented it